inc/thorvg.h

```diff
@@ -394,63 +401,64 @@ class TVG_EXPORT Canvas
      *
-     * @retval @c Result::Success when succeed.
-     * @retval @c Result::MemoryCorruption In case a @c nullptr is passed as the argument.
-     * @retval @c Result::InsufficientCondition An internal error.
+     * @retval Result::Success When succeed.
+     * @retval Result::MemoryCorruption In case a @c nullptr is passed as the argument.
+     * @retval Result::InsufficientCondition An internal error.
      *
-     * @note Rendering order of the paints are in the order of the push. Thus if you mind a layering, sorting the paints before pushing them.
+     * @note The rendering order of the paints is the same as the order as they were pushed into the canvas. Consider sorting the paints before pushing them if you intend to use layering.
      * @see Canvas::reserve()
      * @see Canvas::clear()
      */
     virtual Result push(std::unique_ptr<Paint> paint) noexcept;
 
     /**
      * @brief Sets the total number of the paints pushed into the canvas to be zero.
-     * Depending on the value of the given argument @p free, the Paints were free or not.
+     * Depending on the value of the @p free argument, the paints are freed or not.
      *
-     * @param[in] free If @c then true the memory occupied by paints is deallocated, otherwise it is not.
+     * @param[in] free If @c true, the memory occupied by paints is deallocated, otherwise it is not.
      *
-     * @return @c Result::Success when succeed, @c Result::InsufficientCondition otherwise.
+     * @return Result::Success when succeed, Result::InsufficientCondition otherwise.
      *
-     * @warning If you don't free the paints, they are totally dangled, they are supposed to be reused otherwise you are responsible for their lives. Thus please use that @p free option when you know how it works, otherwise it's not recommended.
+     * @warning If you don't free the paints they become dangled. They are supposed to be reused, otherwise you are responsible for their lives. Thus please use the @p free argument only when you know how it works, otherwise it's not recommended.
      */
     virtual Result clear(bool free = true) noexcept;
 
     /**
      * @brief Request the canvas to update the paint objects.
      *
-     * If a @c nullptr is passed all paint objects retained by the Canvas are updated, otherwise only the paint to which the given @p paint.
+     * If a @c nullptr is passed all paint objects retained by the Canvas are updated,
+     * otherwise only the paint to which the given @p paint points.
      *
      * @param[in] paint A pointer to the Paint object or @c nullptr.
      *
-     * @return @c Result::Success when succeed, @c Result::InsufficientCondition otherwise.
+     * @return Result::Success when succeed, Result::InsufficientCondition otherwise.
      *
-     * @note Updation behavior could be asynchronous by thread option.
+     * @note Update behavior can be asynchronous based on the assigned thread number.
```

Comment:
number is ambiguous...  just async/sync. either thread or not.

Reply:
"The Update behavior can be asynchronous if the assigned thread number is greater than zero.
* @see Initializer::init()"

if you prefer we can stay with 'thread option' although it is not clear to me, which option do we mean:
"The Update behavior can be asynchronous depending on the thread option."

Reply:
@mgrudzinska  ""The Update behavior can be asynchronous if the assigned thread number is greater than zero." this one is much clear.